test/ruleinvalidation.jl

```diff
@@ -42,6 +42,9 @@ end
 # now test invalidation for `inactive`
 inactive(::typeof(issue696), args...) = nothing
 @test autodiff(Forward, issue696, Duplicated(1.0, 1.0))[1] ≈ 0.0
-@test autodiff(Forward, call_issue696, Duplicated(1.0, 1.0))[1] ≈ 0.0
-
+@static if VERSION < v"1.11-"
+    @test_broken autodiff(Forward, call_issue696, Duplicated(1.0, 1.0))[1] ≈ 0.0
+else
+    @test autodiff(Forward, call_issue696, Duplicated(1.0, 1.0))[1] ≈ 0.0
+end
```

Comment:
Yeah that tells me we are doing something wrong again.

Reply:
yeah...........but at least not stupid compile time , but yea.....